setup.py

```diff
@@ -47,7 +47,7 @@
 
 
 setup(
-    name='django-cms',
+    name='cms',
```

Comment:
```suggestion
    name='django-cms',
```

The django-cms package provides two apps: `cms` and `menus`.